docs/product-assembly_pam-on-openshift/src/main/asciidoc/environment-authoring-ha-proc.adoc

```diff
@@ -0,0 +1,42 @@
+[id='environment-authoring-ha-proc']
+= Deploying a high-availability authoring environment
+
+To deploy a high-availability authoring environment, use the `rhpam-70-authoring-ha.yaml` template file. You can download this file from `{URL_BASE_GITHUB_PAM}/templates`.
+
+.Procedure
+
+. Optionally modify the template file:
++
+* To change the database server from PostgreSQL, to MySQL, ???
+* To use an external database server deployed in side ot ourside your OpenShift environment, ???
+* To change the number of instances of {CENTRAL} that are created initially, ???
+
```

Comment:
Are the ??? marks placeholders?

Reply:
They are. I really had to push something out at this point and to try and verify the general process structure. Filling this in is something I really really want to do in the immediately coming days.